spec/services/order_cycle_form_spec.rb

```diff
@@ -260,7 +331,7 @@
           form = OrderCycleForm.new(
             order_cycle,
             { selected_distributor_payment_method_ids: [distributor_payment_method_ii.id] },
-            order_cycle.coordinator
+            order_cycle.coordinator.users.first
           )
```

Comment:
I am a bit confused by this, was the spec wrong before your fix ?

Reply:
Good question. It looks like it was wrong. The form expects a user and, confusingly, the *coordinator* is an enterprise, not a person.

@abdellani It's best to change these things in a separate commit and explain why you are doing this. Then this question doesn't come up in the review.

Reply:
done